content/code-signing/ios-code-signing.md

```diff
@@ -34,45 +34,63 @@ For successful signing, the certificate and the provisioning profile must match
 In short, the purpose of the different provisioning profiles is the following:
 
 - **Development**: for testing the app on a real device while developing. 
-- **Ad Hoc:** for distributing the app to non-TestFlight testers (e.g. via [Testmagic](https://testmagic.io/)). The app must be built in **release** mode.
+- **Ad Hoc:** for distributing the app to non-TestFlight testers. The app must be built in **release** mode.
 - **App Store**: for distributing the app via TestFlight or the App Store. The app must be built in **release** mode.
 
 ## Automatic code signing
 
-Based on the selected provisioning profile type, Codemagic will create a development or a distribution certificate and a development, Ad hoc or App store provisioning profile. The provisioning profile (except for Distribution) will include all the devices you have registered on your Apple Developer account.
+Codemagic makes automatic code signing possible by connecting to the App Store Connect via its API for creating and managing your code signing certificates and provisioning profiles. It is possible to set up several code signing identities and use different code signing settings per workflow.
 
-{{<notebox>}}
-To use automatic code signing, you are required to enable the **Apple Developer portal** integration.
-{{</notebox>}}
+>The following sections describe how to set up automatic code signing for builds configured in the UI. If you're building with `codemagic.yaml`, please refer [here](../yaml/distribution#setting-up-code-signing-for-ios).
```

Comment:
Comma after `codemagic.yaml` looks off:
![Screenshot 2020-08-18 at 16 25 06](https://user-images.githubusercontent.com/2756611/90518396-69bf8500-e16f-11ea-999e-87eb6434ba88.png)

Reply:
Seems to be a styling issue, there's no space between the file name and comma in code. Removing the grey background from the paragraph makes it look slightly better.